config.yaml

```
@@ -439,3 +439,14 @@ steps:
           - id: impc
             weight: 0.2
   ##################################################################################################
+
+
+  #: GO STEP :######################################################################################
+  go:
+    - name: pyspark transform go
+      source:
+        go: input/go/go.obo
+      destination:
+        go: output/go/go.parquet
+      pyspark: go
```

```suggestion
      pyspark: go
      source: input/go/go.obo
      destination: output/go/go.parquet
```

* Move the `pyspark` field up.
* Being only one input and output, you can use `str` instead of `dict[str, str]`.